src/envoy/mixer/BUILD

```diff
@@ -94,6 +90,16 @@ proxy_docker_build(
         "-t",
         "/etc/opt/proxy/envoy.conf.template",
     ],
+    images = [
+        {
+            "name": "proxy",
+            "base": "@docker_ubuntu//:xenial",
+        },
+        {
+            "name": "proxy_debug",
+            "base": "@ubuntu_xenial_debug//file",
+        },
+    ],
     ports = ["9090"],
     repository = "istio",
     tags = ["manual"],
```

Comment:
This PR just re-format the BUILD file.  It did not change this part.  
It should be done by separate PR.